README.md

```diff
@@ -1,21 +1,16 @@
-# Breach Alerts
+# blurts-server
 
 ## Summary
 
-Firefox Breach Alerts notifies users when their credentials have possibly been leaked or stolen in a data breach. Powered by [haveibeenpwned.com](https://haveibeenpwned.com/).
+Firefox Monitor notifies users when their credentials have been compromised in a data breach.
 
-Communicates with the [blurts-addon](https://github.com/mozilla/blurts-addon) client-side add-on for Firefox Monitor.
+This code is for the monitor.firefox.com service & website.
```

Comment:
Why remove the link to the add-on?

Reply:
It doesn't actually communicate with the add-on at all. The add-on just sends users to the site.

Reply:
OK, makes sense. I think I just liked the add-on repo link because it was the only place that I recall that mentioned the add-on portion of the project.
But maybe that will be moot if the add-on code gets moved into tree or not.